src/Symfony/Component/Cache/Traits/RedisTrait.php

```diff
@@ -246,6 +251,9 @@ public static function createConnection($dsn, array $options = [])
         } elseif (is_a($class, \Predis\Client::class, true)) {
             if ($params['redis_cluster']) {
                 $params['cluster'] = 'redis';
+            } elseif ($params['redis_sentinel']) {
```

Comment:
isset(...) (to be consistent)
more importantly: being in and "elseif", this means cluster and sentinel are mutually exclusive? should we throw an exception instead when both are set, to warn ppl their config is not correct?

Reply:
Yes, they definitely don't work with each other. They are two similar but exclusive ways of setting up high availability. Redis Cluster is a sharded approach, Sentinel is single-instance replication.

Exception is probably good.